client/signup/steps/jpo-summary/index.jsx

```diff
@@ -172,7 +172,7 @@ class JPOSummaryStep extends React.Component {
 				<div>
 					<Button primary onClick={ this.completeOnboarding }>
 						{
-							translate( 'Complete Onboarding' )
+							translate( 'Visit your site' )
```

Comment:
Hi! I've found a possible matching string that has already been translated **38** times:
`translate( 'Visit Your Site' )` *ES Score: 11*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).